src/utils/media-utils.js

```diff
@@ -44,6 +46,7 @@ export const resolveUrl = async (url, index) => {
 };
 
 export const guessContentType = url => {
+  if (url.startsWith("webrtc://")) return "video/webrtc";
```

Comment:
I don't think `video/webrtc` is a valid mimetype. Since we probably can't easily know I would do `video/*`

Reply:
going with `video/vnd.hubs-webrtc` for now